bundle/edu.gemini.pot/doc/GhostEstimates.md

```diff
@@ -0,0 +1,612 @@
+
+### ITAC - Reports Updates
+
+_2 Days?_
+
+* Instrument configuration reports
+* Semester statistics report
+
+### ITAC - Queue Engine Update
+
+_2 Days?_
+
+The queue engine just needs to extract the base position instead of the single target coordinates.  Otherwise it needs no changes.  Nevertheless anything that involves editing, compiling, and testing ITAC will take at least a day.
+
+
+# Phase 2 / OT Tasks
+
+### P2 - `Asterism` trait, `SingleTargetAsterism` Implementation
+
+_1 Day_
+
+See details in the proposal above.  Here we just create the new `Asterism` trait and implementation without hooking it up to anything.  This should be able to take advantage of the smallest-circle algorithm developed during the P1 implementation.
+
+### P2 - GHOST Asterisms
+
+_2 Days_
+
+See details in the proposal above.  Here we are defining much of the GHOST model.  This will require a bit of discussion, clarification and iteration.
+
+### P2 - Asterism PIO
+
+_1 Day_
+
+Should use the `ParamSetCodec`.
+
+### P2 - Replace `SPTarget` Base with `Asterism` in `TargetEnv`
+
+This is a big task but here we will mostly concentrate on fairly automatic rote changes.
+
+#### Base Coordinates
+
+_2 Days_
+
+Many cases just need the base position coordinate, and that translates directly
+
+* AGS works with the base position coordinates and shouldn’t have to change much.  The same consideration about sidereal vs. non-sidereal mixed asterisms applies.  I think the right answer is to consider the presence of even one non-sidereal target as counting as non-sidereal. That triggers PWFS to be preferred over any OIWFS.
+
+* ITC needs the base position coordinate in one case, but this translates directly to the asterism base.  `TargetCalculator`, which it uses extensively, is created in terms of base position as well.
+
+* Existing P2 checks look at the science target, but updates should be straightforward.
+
+* Canopus probe ranges depend on distance from base but again this translates to asterism base.  (Of course Canopus isn’t used with dual-targets but in general we have to handle these cases.)
+
+* QV / QPT extracts the RA/Dec of the target but that will become the base position RA/Dec.
+
+* ITAC extracts “rollover” observation information from the ODB.  It uses this to pre-reserve time bins.  This should continue to work if adjusted to use base position coordinates for the asterism. 
+
+
+#### Base Target Name
+
+_1 Day_
+
+Several usages of the name of the target at the base position will have to be replaced.  Presumably a comma-separated list of target names will suffice.
+
+* OT target component uses the name of the target by default.  Some combination of multiple target names can be used when necessary presumably.
+
+* The FITS `OBJECT` header is taken from a sequence param derived from the base position target name.  Again, this presumably can be a comma separated list.
```

Comment:
The space for this keyword is limited to 49 characters (it may be increased if the keyword description is shortened, but only up to 67 characters)